.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!-- Please see CONTRIBUTING.md for guidelines. -->
+
+## Environment
+
+* SuperCollider version:
+* Operating system:
+
```

Comment:
leave the comment about Qt version/other relevant info, that does come in really handy and it's the easiest to forget